apis/bigtable/v1beta1/appprofile_identity.go

```diff
@@ -0,0 +1,117 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1beta1
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/GoogleCloudPlatform/k8s-config-connector/apis/common"
+	refsv1beta1 "github.com/GoogleCloudPlatform/k8s-config-connector/apis/refs/v1beta1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// AppProfileIdentity defines the resource reference to BigtableAppProfile, which "External" field
+// holds the GCP identifier for the KRM object.
+type AppProfileIdentity struct {
+	parent *AppProfileParent
+	id     string
+}
+
+func (i *AppProfileIdentity) String() string {
+	return i.parent.String() + "/appprofiles/" + i.id
+}
+
+func (i *AppProfileIdentity) ID() string {
+	return i.id
+}
+
+func (i *AppProfileIdentity) Parent() *AppProfileParent {
+	return i.parent
+}
+
+type AppProfileParent struct {
+	ProjectID string
```

Comment:
Since "Instance" is the parent for this resource, let's manually update the generated code to something like

```
type AppProfileParent struct {
    BigtableInstance string
}
```

You can find a recent example here, where `NetworkPolicy` is the parent resource of `ExternalAccessRule`.

https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/d70d77e36610be31405e3a7cc5894549705bd772/apis/vmwareengine/v1alpha1/externalaccessrule_identity.go#L45-L47